pkgs/games/steam/fhsenv.nix

```diff
@@ -1,340 +1,124 @@
-{ lib, stdenv, writeShellScript, buildFHSEnv, steam, mesa-demos-i686
-, steam-runtime-wrapped, steam-runtime-wrapped-i686 ? null
-, extraPkgs ? pkgs: [ ] # extra packages to add to targetPkgs
-, extraLibraries ? pkgs: [ ] # extra packages to add to multiPkgs
-, extraProfile ? "" # string to append to profile
-, extraPreBwrapCmds ? "" # extra commands to run before calling bubblewrap (real default is at usage site)
```

Comment:
Could this be reinstated? I need it: https://github.com/Atemu/nixos-config/blob/b705425db59a87afe9681508a0b4e6051be9b0f9/modules/gaming/module.nix#L39-L48 

```patch
diff --git a/pkgs/games/steam/fhsenv.nix b/pkgs/games/steam/fhsenv.nix
index 8b697e25268b..85b0ab83c2db 100644
--- a/pkgs/games/steam/fhsenv.nix
+++ b/pkgs/games/steam/fhsenv.nix
@@ -6,6 +6,7 @@
   extraPkgs ? pkgs: [ ], # extra packages to add to targetPkgs
   extraLibraries ? pkgs: [ ], # extra packages to add to multiPkgs
   extraProfile ? "", # string to append to profile
+  extraPreBwrapCmds ? "", # extra commands to run before calling bubblewrap (real default is at usage site)
   extraBwrapArgs ? [ ], # extra arguments to pass to bubblewrap (real default is at usage site)
   extraArgs ? "", # arguments to always pass to steam
   extraEnv ? { }, # Environment variables to pass to Steam
@@ -105,6 +106,8 @@ let
       "--bind-try /tmp/dumps /tmp/dumps"
     ] ++ extraBwrapArgs;
 
+    inherit extraPreBwrapCmds;
+
     meta =
       if steam != null
       then steam.meta // { mainProgram = name; }
```

Reply:
Oops, got lost somewhere in the refactors. I've put it back now.